redi-connect-front/src/components/atoms/Checkbox.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+import _uniqueId from 'lodash/uniqueId'
+import { Form, Element } from 'react-bulma-components'
+import './Checkbox.scss'
+
+interface Props {
+  name: string
+  value: string
+  checked: boolean
+  children: React.ReactNode
+  disabled: boolean
+  className?: string
+  customOnChange?: Function
+}
+
+// introduce proper types at some point
+function Checkbox (props: any) {
+  const {
+    name,
+    value,
+    checked,
+    disabled,
+    handleChange,
+    handleBlur,
+    isSubmitting,
+    children,
+    customOnChange
+  } = props
+
+  const handleOnChange = customOnChange || handleChange
```

Comment:
Nice use case for nullish coalescing?

Reply:
Handle change will be always defined, customOnChange sometimes. Not sure I understand you here

Reply:
This is the change I have in mind: `customOnChange ?? handleChange`.

It's exactly same the same result - but more explicit.

Result? It'll read like this"
> assign `customOnChange` to `handleOnChange`, unless it's `null` or `undefined`, in which case assign `handleChange` instead

With the current implementation, we instead get:
> ... unless it's falsy ...